dashboard/src/components/DeploymentForm/DeploymentForm.test.tsx

```diff
@@ -131,15 +131,38 @@ it("renders a release name by default, relying in Monickers output", () => {
   expect(name2).toBe("bar");
 });
 
-it("renders the basic form if enabled", () => {
-  const versions = [{ id: "foo", attributes: { version: "1.2.3" } }] as IChartVersion[];
-  const wrapper = shallow(
-    <DeploymentForm
-      {...defaultProps}
-      enableBasicForm={true}
-      selected={{ versions, version: versions[0] }}
-    />,
-  );
-  expect(wrapper.find(LoadingWrapper)).not.toExist();
-  expect(wrapper.find(AceEditor)).not.toExist();
```

Comment:
Looks like we should have tests which show that the form is rendered without tabs if the new feature is not enabled?